packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/ovirt/ovirtSecretFieldValidator.ts

```diff
@@ -17,8 +17,8 @@ import {
  * 'warning' - The field's value has passed validation but does not fit the standard format, it's the user's choice if to accept that value.
  * 'error' - The field's value has failed validation.
  */
-export const ovirtSecretFieldValidator = (id: string, value: string): ValidationMsg => {
-  const trimmedValue = value.trim();
+export const ovirtSecretFieldValidator = (id: string, value: string | boolean): ValidationMsg => {
```

Comment:
see comment above about value should not be boolean
https://github.com/kubev2v/forklift-console-plugin/pull/1117/files#r1568396476

Reply:
I changed it, see comment above